java/integTest/src/test/java/glide/modules/BasicTests.java

```diff
@@ -0,0 +1,22 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.modules;
+
+import static glide.TestUtilities.commonClusterClientConfig;
+import static glide.api.models.configuration.RequestRoutingConfiguration.SimpleSingleNodeRoute.RANDOM;
+
+import glide.api.GlideClusterClient;
+import glide.api.models.commands.InfoOptions.Section;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+
+// TODO: remove when real tests added
+public class BasicTests {
+    @Test
+    @SneakyThrows
+    public void connection_check() {
+        var client =
+                GlideClusterClient.createClient(commonClusterClientConfig().requestTimeout(5000).build())
+                        .get();
+        System.out.println(client.info(new Section[] {Section.MODULES}, RANDOM).get().getSingleValue());
```

Comment:
Does this contain an interesting value for JSON or VSS?

Reply:
It gives us an option to see which modules are installed.

Reply:
Cool.  If we make this file tests for VectorSearch, can we check that the value contains "VSS" or something similar?  And do the same for JSON?

Reply:
yes

Reply:
Maybe use a `@beforeall` to set a variable and we can use `org.junit.Assume` for all the tests if true (so, ignore all tests if the module isn't loaded...?)

Reply:
We won't see failure with `assume`. Will add an assert in a follow up PR.